public/script.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-undef */
+
+const articleHeadings = document.querySelectorAll('h2[data-itemtype="http://schema.stenci.la/Heading"], h2[itemtype="http://schema.stenci.la/Heading"], h2.evaluation-summary__header');
+
+let throttleTimer;
+
+const throttle = (callback, time) => {
+  if (throttleTimer) return;
+  throttleTimer = true;
+  setTimeout(() => {
+    callback();
+    throttleTimer = false;
+  }, time);
+};
+
+function navHighlighter() {
+  const scrollY = window.pageYOffset;
+
+  for (let i = 0; i < articleHeadings.length; i += 1) {
+    const current = articleHeadings[i];
+    const next = articleHeadings[i + 1];
+    const sectionTop = current.offsetTop - 50;
+    const sectionId = current.getAttribute('id');
+
+    if (
+      scrollY > sectionTop
```

Comment:
I think if we also check for this being the first element (`(scrollY > sectionTop || i === 0)`) we can highlight the first header until it's scrolled past?

Reply:
This could cause flickering when you scroll, it would activate the top one before activating the next.